packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -40,7 +41,11 @@ export function isUserManagementEnabled(): boolean {
 }
 
 export function isSharingEnabled(): boolean {
-	return isUserManagementEnabled() && config.getEnv('enterprise.features.sharing');
+	const license = getLicense();
+	return (
+		isUserManagementEnabled() &&
+		(config.getEnv('enterprise.features.sharing') || license.isSharingEnabled())
```

Comment:
I suppose `config.getEnv('enterprise.features.sharing')` will ultimately be removed?

Reply:
I doubt it.. I think this is how we are enabling features on cloud.. using hooks to set this to true